ui/src/components/Drawer/index.vue

```diff
@@ -1,15 +1,22 @@
 <template>
   <n-drawer
+    id="drawer-inner-target"
     resizable
     placement="right"
     :show="showDrawer"
     :min-width="drawerMinWidth"
     :max-width="drawerMaxWidth"
-    :default-width="drawerDefaultWidth"
-    @update:show="closeDrawer"
+    :width="drawerDefaultWidth"
+    @mask-click="closeDrawer"
   >
     <n-drawer-content closable :native-scrollbar="false" :header-style="DRAWER_HEADER_STYLE">
-      <n-tabs size="small" type="bar" :value="contentType" :pane-style="{ marginTop: '10px' }" @update:value="handleChangeTab">
+      <n-tabs
+        size="small"
+        type="bar"
+        :value="contentType"
+        :pane-style="{ marginTop: '10px' }"
+        @update:value="handleChangeTab"
+      >
         <n-tab-pane name="setting" display-directive="if" :tab="t('Settings')">
           <Setting :settings="settingsConfig" />
         </n-tab-pane>
```

Comment:
There's no information provided on what needs to be checked regarding the code. I need more context about which specific parts of your template and components you want reviewed for irregularities, potential issues, or if there are any suggestions for optimization.

If it's referring to CSS styles (e.g., `DRAWER_HEADER_STYLE`) within `n-drawer` component, then it should refer either to local CSS or global styles like Tailwind classes such as `:native-scrollbar`.

For other templates and components, please give more detailed information on what aspects need reviewing so that relevant insights can be offered effectively.